salalib/tidylines.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include "salalib/spacepix.h"
+
+// helpers... a class to tidy up ugly maps people may give me...
+
+class TidyLines : public SpacePixel
+{
+public:
+   TidyLines() {;}
+   virtual ~TidyLines() {;}
```

Comment:
Is some other class derived from `TidyLines`? Or why do we need an empty virtual destructor?